### PR TITLE
feat(doc): document subteam details

### LINKED ISSUE #215: team info for the README
team leads @skade @booyaa @sebasmagri @jonathandturner @Manishearth please comment with a link to your teams repo (if applicable), call for new members, goals document, meeting schedule

i'll be making a PR to update the README in this repo with all the info :D

Please also add a 2-3 sentence explanation of what your team does.

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #215

### CODE REVIEW

**Comment by Manishearth** [comment]
The switchboard channel is IRC. Our goals doc has moved to the readme: https://github.com/rust-community/switchboard-team/blob/master/README.md

File: README.md [hunk 1]
```diff
@@ -1,9 +1,87 @@
...
+
+### 📚 Content
+
+Organize and write for the community blog and other publication outlets, Organise and promote material like videos and podcasts, Provide lectorate (copy editing) and assistance for people that want to write blog posts on rust
+
+[**Repo**](https://github.com/rust-community/content-team) 
+ | [**Goals Document**](https://docs.google.com/document/d/1fNKaOiV-D2VHdNUPJoGFsM2D10RWNsTGj5WjGHLboqg)
+| **Leader**: Mark Sta Ana ([@booyaa])
+
+### ☎️ Swtichboard
```
Comment (Manishearth) [inline README.md]: speling

Comment (ashleygwilliams) [reply README.md]: fixed and fixed up into the subteam commit

File: README.md [hunk 2]
```diff
@@ -1,9 +1,87 @@
...
+### 🌉  RustBridge
+
+RustBridge is a workshop focused on getting underrepresented people with a background in another programming language to learn Rust and join the community.
+
+[**Repo**](https://github.com/rustbridge/team)
+| **Leader**: Ashley Williams ([@ashleygwilliams])
+
+### 📝  Survey
+
+Plan, run, analyse, publish community and commercial surveys
```
Comment (jonathandturner) [inline README.md]: Focus is on the community side more than the commercial side.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,87 @@
...
+[**Repo**](https://github.com/rust-community/events-team/)
+ | [**Goals Document**](https://docs.google.com/document/d/1jH2Cz493ILQ79mTR1O8Msgf4v7UmaYp5Mc0UjTNmQ68/edit#)
+ | **Leader**: Florian Gilcher ([@skade])
+
+### 📚 Content
+
+Organize and write for the community blog and other publication outlets, Organise and promote material like videos and podcasts, Provide lectorate (copy editing) and assistance for people that want to write blog posts on rust
+
+[**Repo**](https://github.com/rust-community/content-team) 
+ | [**Goals Document**](https://docs.google.com/document/d/1fNKaOiV-D2VHdNUPJoGFsM2D10RWNsTGj5WjGHLboqg)
```
Comment (booyaa) [inline README.md]: Just added our goals to the repo, could you please reference this instead? https://github.com/rust-community/content-team/blob/master/GOALS.md

File: README.md [hunk 4]
```diff
@@ -1,9 +1,87 @@
...
+
+We are hoping to find new members around the globe, especially outside of Europe and the US.
+
+[**Repo**](https://github.com/rust-community/events-team/)
+ | [**Goals Document**](https://docs.google.com/document/d/1jH2Cz493ILQ79mTR1O8Msgf4v7UmaYp5Mc0UjTNmQ68/edit#)
+ | **Leader**: Florian Gilcher ([@skade])
+
+### 📚 Content
+
+Organize and write for the community blog and other publication outlets, Organise and promote material like videos and podcasts, Provide lectorate (copy editing) and assistance for people that want to write blog posts on rust
```
Comment (booyaa) [inline README.md]: Whoops, editing the focus statement fix typos should be:

> Organize and write for the community blog and other publication outlets; organize and promote material like videos and podcasts; provide lectorate (copy editing) and assistance for people that want to write blog posts on rust.

Also updated the typo in the repo.

**Comment by ashleygwilliams** [comment]
@jonathandturner @booyaa lemme know if those changes look good!

**Pull request merged**